cmake/features.cmake

```diff
@@ -135,7 +135,8 @@ set(IE_EXTRA_MODULES "" CACHE STRING "Extra paths for extra modules to include i
 ie_dependent_option(ENABLE_TBB_RELEASE_ONLY "Only Release TBB libraries are linked to the Inference Engine binaries" ON "THREADING MATCHES TBB;LINUX" OFF)
 
 get_linux_name(LINUX_OS_NAME)
-if(LINUX_OS_NAME MATCHES "^Ubuntu [0-9]+\.[0-9]+$" AND NOT DEFINED ENV{TBBROOT})
+if(LINUX_OS_NAME MATCHES "^Ubuntu [0-9]+\.[0-9]+$"
+   AND NOT DEFINED ENV{TBBROOT} AND NOT DEFINED ENV{TBB_DIR})
```

Comment:
`ENABLE_SYSTEM_LIBS_DEFAULT` is used for both TBB and Pugixml. Can we define `ENABLE_SYSTEM_LIBS_DEFAULT` only based on Ubuntu, while create ENABLE_SYSTEM_TBB_DEFAULT which takes into account env vars?

Reply:
btw, does it fix CVS-89248?

Reply:
See https://github.com/openvinotoolkit/openvino/pull/12534